### DISCUSSION TITLE
angular-three does not work with angular16+

### DISCUSSION THREAD

**Discussion by Reiserro** [angular-threejs/angular-three#25] [General]
i want to include angular-three in my angular project. however, i get the error that the module is no longer angular16+ compatible. are there any other options besides going to a lower angular version?  --force, or --legacy-peer-deps dont work

![error](https://github.com/angular-threejs/angular-three/assets/48568613/911acfd2-b128-4683-9d14-8bd2145b7dae)

**Comment by IamSarki** [comment]
I tried to brute force the installation with angular 17 and then use this [example](https://stackblitz.com/edit/angular-three-demo-template-gbtvkp?file=src%2Fapp%2Fapp.component.ts) from the tutorial and it actually loads without problem. I am trying with other examples just to see if there are any problems.
I used this:
`npm i angular-three --legacy-peer-deps`
`npm i three --legacy-peer-deps`
`npm i -D @types/three --legacy-peer-deps`
`npm i three-stdlib --legacy-peer-deps`
`skipLibCheck: true in tsconfig.json`
However, I don't know what problems this could create.

**Reply by MitchSmoot** [reply]
Angular 17 support was slated for the angular-three 2.0 release. It's a big update, adding signal support, and the signal and control flow changes in 17 have caused angular-three 2.0 to take longer than expected.

**Reply by IamSarki** [reply]
I see, thanks. And I didn't notice the branch 'platform-v17', the main ReadMe should be much more informative in my opinion.
